src/vstart.sh

```diff
@@ -736,6 +742,38 @@ $extra_conf
 EOF
 }
 
+write_logrotate_conf() {
+    out_dir=$(pwd)"/out/*.log"
+
+    cat << EOF
+$out_dir
+{
+    copytruncate
```

Comment:
add `notifempty` otherwise empty client logs get continually rotated if `-f` option is used

Reply:
Added.

Reply:
Added.